testing/endtoend/components/beacon_node.go

```diff
@@ -283,7 +283,7 @@ func (node *BeaconNode) Start(ctx context.Context) error {
 	// on our features or the beacon index is a multiplier of 2 (idea is to split nodes
 	// equally down the line with one group having feature flags and the other without
 	// feature flags; this is to allow A-B testing on new features)
-	if !config.TestFeature || index%2 == 0 {
+	if !config.TestFeature || index != 1 {
```

Comment:
The comment above indicates this was supposed to be only for multiples of 2 - which one is intended?

Reply:
Will fix the comment, but the current intent is to only have 1 node running as a control rather than splitting it down the line